examples/todolist/src/router.ts

```diff
@@ -72,7 +75,18 @@ export const router = createRouter({
         },
       },
     } as const,
-    handler: async request => {
+    handler: async (
+      request: TypedRequest<
```

Comment:
```suggestion
      request: TypedRequest<
```
This shouldn't be like that. It should be inferred from `schema` above. It is not given explicitly on purpose.

Reply:
Well, I'm not sure where typescript error coming from... I had found this way to fix it. Can you please look yourself?

Reply:
I've reverted those types

Reply:
@ardatan please approve